roles/openshift_health_checker/test/fluentd_test.py

```diff
@@ -103,7 +100,7 @@ def assert_error(error, expect_error):
     ),
 ])
 def test_get_fluentd_pods(pods, nodes, expect_error):
-    check = canned_fluentd(exec_oc=lambda cmd, args: json.dumps(dict(items=nodes)))
-
+    check = fluentd()
```

Comment:
Hmm, wouldn't it be way simpler to just write `Fluentd()` (with an uppercase F) here?! What's the point of the fixture?